frontend/src/Frontend/UI/Wallet.hs

```diff
@@ -379,6 +379,8 @@ accountGuardSummary (AccountGuard_Other pactGuard) =
   where
     gType = pactGuardTypeText $ Pact.guardTypeOf pactGuard
 
+accountGuardSummary (AccountGuard_KeySetRef name) = "keyset-ref: " <> name
```

Comment:
Let's just say "ref: " here.  A lot of times the widgets that call `accountGuardSummary` are space-constrained.